test/jump.jl

```diff
@@ -1,6 +1,6 @@
-jump_01vec(optimizer, mode = BilevelJuMP.SOS1Mode(), config = Config()) = _jump_01(optimizer, true, mode, config)
-jump_01(optimizer, mode = BilevelJuMP.SOS1Mode(), config = Config()) = _jump_01(optimizer, false, mode, config)
-function _jump_01(optimizer, vectorized::Bool, mode, config)
+jump_01vec(optimizer, mode = BilevelJuMP.SOS1Mode(), consider_constrained_variables = consider_constrained_variables, config = Config()) = _jump_01(optimizer, true, mode, consider_constrained_variables, config)
```

Comment:
this should be a field in config.
Also, I am not sure this should be run in all tests.

Reply:
Left this for now, as I'm not sure which tests to select best etc. 
Personally, I don't think that running tests twice adds a huge amount of time, as almost everything should be compiled already and the problems are fairly small. 
On my machine, running tests like this adds about 6 seconds (less than 7%)

Reply:
But I agree a field in config would be better, will change this...

Reply:
I could also do the subset, but I'm not sure which ones to select...
Do you have a preference?

Reply:
if the overhead is only 7% for running twice, this would indeed be fine